lib/prom_ex/grafana_client.ex

```diff
@@ -18,6 +18,7 @@ defmodule PromEx.GrafanaClient do
     401 => :unauthorized,
     403 => :forbidden,
     404 => :not_found,
+    409 => :already_exists
```

Comment:
Missing a comma here

Reply:
yeah sorry i forgot to force push that fix :)